src/writing-policies/rust.md

```diff
@@ -1,25 +1,647 @@
-> **Note well:** WebAssembly and WASI are fast evolving targets. The contents
-> of this page have been written during Nov 2020, hence they could be outdated.
->
-> Please open an issue if the contents of this page have become outdated.
-
 # Rust
 
 [Rust](https://www.rust-lang.org/) is the most mature programming language that
-can generate WebAssembly modules.
+can generate WebAssembly modules: WebAssembly is a first-class citizen
+in the Rust world. That means many of the tools and crates of the Rust
+ecosystem work out of the box.
+
+Kubewarden provides a [Rust SDK](https://crates.io/crates/kubewarden-policy-sdk)
+that simplifies the process of writing policies, plus a
+[template project](https://github.com/kubewarden/policy-rust-template) to
+quickly scaffold a policy project.
```

Comment:
I would mention that the template project should be used with `cargo-generate` -- just in case the reader wants to open that right away

Reply:
Done